lib/logster/base_store.rb

```diff
@@ -125,13 +139,24 @@ def report(severity, progname, msg, opts = {})
         message.backtrace = caller.join("\n")
       end
 
-      return if ignore && ignore.any? { |pattern| message =~ pattern }
+      return if ignore && ignore.any? do |pattern|
+        if message =~ pattern
+          val = Regexp === pattern ? pattern.inspect : pattern.to_s
```

Comment:
this is a tiny bit odd to have a String / Regex mashup, not a huge deal, but we should be consistent internally imo.